auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -109,14 +115,46 @@ def get_bugs(self, date="today", bug_ids=[], chunk_size=None):
                     elif new_access_tag == copied_fields["whiteboard"]["value"]:
                         copied_fields["whiteboard"]["from"].append(dup_bug["id"])
 
+                # Regressed by: move the regressed_by field to the duplicate of
+                if dup_bug["regressed_by"]:
+                    # We always want to remove regressed_by from duplicate bugs
```

Comment:
When we'll send the announcement, we should gather feedback about this point

Reply:
This point is tricky. There is no guarantee that both transactions (remove from the duplicate and add to the duplicate of) will be successful.

We could implement it differently by modifying the regressions field on the regressor bug to add and remove in the same transaction, but that will be noisy.

Reply:
As we discussed in our last meeting, we will only copy the values from the `regressed_by` field. Done in f80a9c8d99ed27bee22a3c620e3dd5e11e0ffa65.